cmd/create/cluster/cmd.go

```diff
@@ -644,6 +649,14 @@ func run(cmd *cobra.Command, _ []string) {
 		isIAM = !isSTS
 	}
 
+	isSTS = isSTS || awsCreator.IsSTS
```

Comment:
NP
Maybe we can append `|| awsCreator.IsSTS` to line `630`?
To have all the scenarios in one place.

Reply:
I wasn't sure why this was after actually, maybe there's a reason. But if you are okay with it as well I'll gladly change it

Reply:
Oh, I thought you've added it here.
Now I saw the deletion, never mind.